app/assets/stylesheets/layout/helper-classes.scss

```diff
@@ -220,58 +220,58 @@
 
 /* ==================== 6. Background Colours ==================== */
 .whitebg {
-  background-color: $white;
+  background-color: $white !important;
 }
 
 .greybg {
-  background-color: $stone;
+  background-color: $stone !important;
 }
 
 .light-greybg {
-  background-color: $ash;
+  background-color: $ash !important;
 
   &--small-only {
     @include breakpoint(small only) {
-      background-color: $ash;
+      background-color: $ash !important;
     }
   }
 }
 
 .insurance-blue-bg {
-  background-color: $brand-blue-insurance;
+  background-color: $brand-blue-insurance !important;
 }
 
 .light-blue-bg {
-  background: $brand-blue-light;
+  background: $brand-blue-light !important;
 }
 
 .dark-blue-bg {
-  background: $brand-blue-dark;
+  background: $brand-blue-dark !important;
 }
 
 .rewards-green-bg {
-  background-color: $brand-green;
+  background-color: $brand-green !important;
 }
 
 .orange-bg {
-  background-color: $orange;
+  background-color: $orange !important;
 }
 
 .red-bg {
-  background: $brand-red;
+  background: $brand-red !important;
```

Comment:
I've noticed some inconsistencies with the attribute we are using to apply background color. Some classes use background, and some use background-color. Could we make these consistent?